aws_helper/config.go

```diff
@@ -124,7 +124,7 @@ func CreateAwsSession(config *AwsSessionConfig, terragruntOptions *options.Terra
 }
 
 // Make API calls to AWS to assume the IAM role specified and return the temporary AWS credentials to use that role
-func AssumeIamRole(iamRoleArn string) (*sts.Credentials, error) {
+func AssumeIamRole(iamRoleArn string, duration int64) (*sts.Credentials, error) {
```

Comment:
NIT: Could you use a name that makes the units clearer? E.g., `durationSeconds`?

Reply:
Renamed it to `sessionDurationSeconds` which is clearer in terms of what the duration is for, as well as the unit. Thanks for the feedback.